### PR TITLE
docs(CORS): Reminder to set CORS headers in code when using lambda-proxy

### PR DESCRIPTION
**Pull request by wedgybo**
## What did you implement:

Update to docs regarding CORS

## Todos:

- [x] Write documentation

***Is this ready for review?:*** YES

### CODE REVIEW

**Comment by wedgybo** [comment]
@flomotlik Updated

File: docs/02-providers/aws/events/01-apigateway.md [hunk 1]
```diff
@@ -422,6 +424,20 @@ functions:
 
 This example is the default setting and is exactly the same as the previous example. The `Access-Control-Allow-Methods` header is set automatically, based on the endpoints specified in your service configuration with CORS enabled.
 
+**Note:** If you are using the default lambda proxy integration. Remember to include `Access-Control-Allow-Origin` in your returned headers object otherwise CORS will fail.
```
Comment (flomotlik) [inline docs/02-providers/aws/events/01-apigateway.md]: ". Remember" should probably be ", remember"?

File: docs/02-providers/aws/events/01-apigateway.md [hunk 2]
```diff
@@ -90,6 +90,8 @@ exports.handler = function(event, context) {
 };
 ```
 
+**Note:** If you want to use CORS with the lambda-proxy integration. Remember to include `Access-Control-Allow-Origin` in your returned headers object.
```
Comment (flomotlik) [inline docs/02-providers/aws/events/01-apigateway.md]: ". Remember" should probably be ", remember"?

**Comment by flomotlik** [comment]
Sorry missed the typo on first read, after the typo we're good. Thanks for this and adding an example as well!

**Pull request merged**